src/components/Sidebar/Sidebar.jsx

```diff
@@ -60,18 +60,14 @@ export const Sidebar = ({
     const hideRightDrawer = rightDrawer?.count <= 0 || !rightDrawer;
 
     const handleOnClose = (direction, closeDrawer) => {
-        if (direction === "left") {
-            if (closeDrawer) {
+        if (closeDrawer) {
```

Comment:
call the `.onClose` method only on actual drawer close (e.g. avoid calling when click inside filter popover), also refactored `if` clauses according to the linter to avoid lonely if inside else block